src/spaceObjects/scanProbe.cpp

```diff
@@ -49,7 +49,7 @@ void ScanProbe::update(float delta)
 
 bool ScanProbe::canBeTargetedBy(P<SpaceObject> other)
 {
-    return (getTarget() - getPosition()) < getRadius();
+    return true;
```

Comment:
This change is problematic for the AI. This (slightly odd) logic is in place to prevent AI ships from actively seeking out probes and destroying them. But only seek and destroy probes when they are near the ships.

Reply:
Ok, I did'nt think about that. I think I did this to select and active the 3D vision just when the probe is launched. So I am not sure this feature is relevant anymore and that the relay officier have to active camera manually.

Reply:
Hi,
I have another way to solve this: 
I use the Selectable criteria instead of Targetable, and I make all probes selectable by any object of the same faction (which makes sense to me). so targetable is left with the old slightly odd logic.
see:
https://github.com/amir-arad/EmptyEpsilon/commit/11c201fcffab094c4ab86babc249aa5485594c91